### PR TITLE
Updated list of public ensembl servers. Updated public metadata datab…

### PR DESCRIPTION
**Pull request by thomasmaurel**
…ase name to match versionned databases. Fix bug in lookup where method was receiving more than one database adaptors

### CODE REVIEW

File: modules/Bio/EnsEMBL/LookUp/RemoteLookUp.pm [hunk 1]
```diff
@@ -258,15 +258,15 @@ sub get_all_by_taxon_id {
...
+	Description : Return all database adaptors that has the supplied string as an alias/name
 	Argument    : String
 	Exceptions  : None
 	Return type : Arrayref of Bio::EnsEMBL::DBSQL::DatabaseAdaptor
 =cut
 
 sub get_by_name_exact {
   my ( $self, $name ) = @_;
-  return $self->genome_to_dba( $self->adaptor()->fetch_by_any_name($name) );
+  return $self->genomes_to_dbas( $self->adaptor()->fetch_by_any_name($name) );
```
Comment (marcoooo) [inline modules/Bio/EnsEMBL/LookUp/RemoteLookUp.pm:269]: I hope nobody is using the method apart from us, it's quite a change in API contract.

File: modules/Bio/EnsEMBL/MetaData/DBSQL/GenomeInfoAdaptor.pm [hunk 2]
```diff
@@ -132,7 +133,8 @@ use Bio::EnsEMBL::MetaData::DBSQL::MetaDataDBAdaptor;
 sub build_ensembl_adaptor {
   my ( $self, $release ) = @_;
   my $args = e_args();
-  $args->{-DBNAME} = 'ensembl_metadata';
+  my $sofware_version=software_version();
+  $args->{-DBNAME} = 'ensembl_metadata_'.$sofware_version;
```
Comment (marcoooo) [inline modules/Bio/EnsEMBL/MetaData/DBSQL/GenomeInfoAdaptor.pm:137]: The new version of the metadata is not supposed to be versioned anymore :-)

File: modules/Bio/EnsEMBL/LookUp/RemoteLookUp.pm (hunk 1 above)
Comment (thomasmaurel) [reply modules/Bio/EnsEMBL/LookUp/RemoteLookUp.pm:269]: @marcoooo: Well the method is broken at the moment so I guess nobody was using it... It's again related to the change made to support species in multiple division, e.g: drosophila.

File: modules/Bio/EnsEMBL/MetaData/DBSQL/GenomeInfoAdaptor.pm (hunk 2 above)
Comment (thomasmaurel) [reply modules/Bio/EnsEMBL/MetaData/DBSQL/GenomeInfoAdaptor.pm:137]: Well then we will revert back...

File: modules/Bio/EnsEMBL/LookUp/RemoteLookUp.pm (hunk 1 above)
Comment (marcoooo) [reply modules/Bio/EnsEMBL/LookUp/RemoteLookUp.pm:269]: OK :+1:

File: modules/Bio/EnsEMBL/MetaData/DBSQL/GenomeInfoAdaptor.pm (hunk 2 above)
Comment (marcoooo) [reply modules/Bio/EnsEMBL/MetaData/DBSQL/GenomeInfoAdaptor.pm:137]: we'll surely get rid of the PERL part, anyway ;-)

**Comment by kamaldodiya** [comment]
As such I have nothing much to say but I was following the document available on the site http://www.ensembl.org/info/docs/Doxygen/metadata-api/classBio_1_1EnsEMBL_1_1LookUp.html (Species_name)
http://www.ensembl.org/info/docs/Doxygen/metadata-api/classBio_1_1EnsEMBL_1_1LookUp_1_1RemoteLookUp.html (method name build_ensembl_genomesadaptor() actually it is  build_ensembl_genomes_adaptor();)
There may be more. If someone is following this doc then they will run into problem. Is there any plan to update the docs ?

**Comment by thomasmaurel** [comment]
> As such I have nothing much to say but I was following the document available on the site http://www.ensembl.org/info/docs/Doxygen/metadata-api/classBio_1_1EnsEMBL_1_1LookUp.html (Species_name)
> http://www.ensembl.org/info/docs/Doxygen/metadata-api/classBio_1_1EnsEMBL_1_1LookUp_1_1RemoteLookUp.html (method name build_ensembl_genomesadaptor() actually it is build_ensembl_genomes_adaptor();)
> There may be more. If someone is following this doc then they will run into problem. Is there any plan to update the docs ?

@kamaldodiya: Thanks, I will update the documentation on this PR.

**Pull request merged**